src/components/plots/styling/MarkerGeneSelection.jsx

```diff
@@ -43,7 +46,15 @@ const MarkerGeneSelection = (props) => {
 
     return (
       <Space direction='vertical' size='small'>
-        <p style={{ margin: 0 }}>Type in a gene name and select it to add it to the plot.</p>
+        <p style={{ margin: 0 }}>
+          Type in a gene name and select it to add it to the plot.
+          <Tooltip
+            title='To add multiple genes, separate them with a space or comma.'
+          >
+            &nbsp;
+            <QuestionCircleOutlined />
+          </Tooltip>
```

Comment:
I think this information should be placed together with the description, not in a tooltip.

Reply:
Any reason to allow **both** space and comma to delimit gene names? Why not just use comma? I think it'll be easier for us to handle and less ambiguous for users.

Reply:
This is to allow two use cases:
- searching for genes individually (where separating with a comma seems more natural)
- pasting a list of genes from somewhere (separated by whitespaces)

It might make more sense to have them only separated by spaces, but users might run into an edge case with genes with multiple IDs which have format `name - ENSG#` while searching. That's also what the regex is for.

I didn't want the description to be too cluttered so I put it in a tooltip, but I will try to rewrite it to be consice and also include the tooltip info.

Reply:
I didn't want the description to be too cluttered so I put it in a tooltip, but I will try to rewrite it to be consice and also include the tooltip info.